packages/vuetify/build/webpack.dev.config.js

```diff
@@ -68,10 +68,10 @@ module.exports = merge(baseWebpackConfig, {
     ]
   },
   devServer: {
-    static: resolve('../dev'),
+    // static: resolve('../dev'),
     host: process.env.HOST || 'localhost',
     port: process.env.PORT || '8080',
-    firewall: false,
+    // firewall: false,
```

Comment:
Update to the new webpack-dev-server schema

Reply:
I wasn't able to find any information regarding this in webpack-dev-server@3.x, only 4.0-rc. Could you clarify this?